src/stcal/dark_current/dark_class.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, science_model=None):
 
             self.exp_nframes = science_model.meta.exposure.nframes
             self.exp_groupgap = science_model.meta.exposure.groupgap
+            self.exp_intstart = science_model.meta.exposure.integration_start
```

Comment:
This fails RomanCAL regression tests. There is no 'integration_start' in the roman exposure group.  Maybe Try/except this line?

Reply:
Good catch. Fixed in latest commit.

Reply:
@PaulHuwe Could you test again?